src/App.css

```diff
@@ -0,0 +1,91 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+
+body{
+    background-color: #ffc74a;
+}
+header {
+     height: auto;
+     width: auto;
+     margin-bottom: 30px;
+ }
+
+.logo{  
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    height:auto;
+    width:25%;
+    position: relative;
+}
+.pageButtons{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+}
+.pageButton1, .pageButton2{
+    background: #004E99;
+    color: white;
+    font-family: "Montserrat", serif;
+    font-weight: bolder;
+    padding: 10px 22px;
+    margin: 20px;
+    cursor:pointer ;
+    font-size: 14px;
+    width: 110px;
+}
+
+.App {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+
+button:disabled{
+    opacity: 20%
+}
+
+.pokemonCard {
+    padding: 10px;
+    border: 1px solid #004E99;
+    background-color: white;
+    text-align: center;
+    flex:18%;
```

Comment:
De flex property is een shorthand property voor: flex-grow, flex-shrink en flex-basis. Die verwacht drie waardes, zoals bijvoorbeeld 0 1 auto waarin geen procenten worden gebruikt